communication/slack-config/channels.yaml

```diff
@@ -345,6 +345,7 @@ channels:
     archived: true
   - name: wg-app-def
     archived: true
+  - name: wg-api-expression
   - name: wg-apply
```

Comment:
Can the `wg-apply` channel be archived now that we would have `wg-api-expression`?